vignettes/quickstart.Rmd

```diff
@@ -211,7 +211,11 @@ anaObj
 
 Note that, similar to `draws()`, the `ancova()` function uses the `set_vars()`
 function which determines the names of the key variables within the data and the covariates 
-(in addition to the treatment group) for which the analysis model will be adjusted. 
+(in addition to the treatment group) for which the analysis model will be adjusted.
+
+The results from `ancova` are collapsed into a single list suffixed by the visit name.
```

Comment:
I don't think we need to mention about the list here as users shouldn't be directly interacting with it. The user interaction point is  `as.data.frame(pool_object)`